lib/dx/ecto/query.ex

```diff
@@ -485,6 +486,10 @@ defmodule Dx.Ecto.Query do
     from(q in queryable, order_by: ^fields)
   end
 
+  def sleep(queryable, duration) do
```

Comment:
Can you think of a less invasive way of triggering the timeout for tests?

Reply:
Yes, configuring a custom test loader that triggers the timeout should work.

Reply:
I don't understand how implementing a custom test loader helps us test the timeout behaviour of the default loader.

Maybe I misunderstood.